extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -80,18 +80,20 @@ class AmpLightbox extends AMP.BaseElement {
     this.requestFullOverlay();
     this.getViewport().resetTouchZoom();
     this.getViewport().hideFixedLayer();
-    this.element.style.display = '';
-    this.element.style.opacity = 0;
 
-    // TODO(dvoytenko): use new animations support instead.
-    this.element.style.transition = 'opacity 0.1s ease-in';
-    requestAnimationFrame(() => {
-      this.element.style.opacity = '';
+    this.mutateElement(() => {
```

Comment:
I described the possible problem in the bug. But reviewing again - it doesn't apply here because the only thing that's animated is opacity. If we animated other properties we might have had an issue. As it stands - this is fine to leave as is.